src/pdf-viewer.scss

```diff
@@ -0,0 +1,30 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-pdf-viewer;
+
+.#{$block} {
+  &__dialog {
+    max-height: calc(100vh - 8rem);
+  }
+
+  &__title {
```

Comment:
Dialog is using Bar by default (as specification says) and I think we should still use Bar component to create this layout.
Current Dialog Header/Sub-header implementation provides all necessary styling and blocks for building the PDF Viewer Header.

For sure there is no reason to create PDF Viewer specific styling to make Bar look like Toolbar in transparent mode.